builtin/provisioners/chef/linux_provisioner.go

```diff
@@ -10,10 +10,27 @@ import (
 )
 
 const (
-	chmod      = "find %s -maxdepth 1 -type f -exec /bin/chmod %d {} +"
+	chmodLinux = "find %s -maxdepth 1 -type f -exec /bin/chmod %d {} +"
+	chmodAix   = "find %[1]s ! -path %[1]s -prune -type f -exec /bin/chmod %[2]d {} +"
 	installURL = "https://omnitruck.chef.io/install.sh"
 )
 
+func (p *provisioner) getChmodCommand() string {
+	if p.OSType == "aix" {
+		return chmodAix
+	} else {
+		return chmodLinux
+	}
+}
+
+func (p *provisioner) getRootGroup() string {
+	if p.OSType == "aix" {
+		return "system"
+	} else {
+		return "root"
+	}
```

Comment:
My linter gives a warning about this as the else is not needed here:

```
	if p.OSType == "aix" {
		return "system"
	}
	return "root"
```

Reply:
I do like that much better. Will fix and push up.